qa/integration/services/helpers.sh

```diff
@@ -2,6 +2,7 @@
 set -e
 current_dir="$(dirname "$0")"
 
+BIN_DIR=$current_dir/../../../bin
```

Comment:
This bin dir appears to be a sibling to the root of our repo, and I know at least on my Dev machine i don't satisfy this requirement.

Can/should our CI setup prepend this bin directory to its `PATH`, so when we go to execute `ruby` below it is found using the normal methods?

Reply:
The intent for this was to use the [ruby script](https://github.com/elastic/logstash/blob/master/bin/ruby) supplied by Logstash -the relative path when this script is run in the context of a build is intended to be `$LS_HOME/bin`, which wasn't captured well by the variable name in this PR. 

From a test run I did on the build box to ensure this works: https://logstash-ci.elastic.co/job/elastic+logstash+pull-request+multijob-integration-1/1592/console:

```
17:58:14 + test_port_ruby 9092
17:58:14 + /opt/logstash/qa/integration/services/../../../bin/ruby -rsocket -e 'TCPSocket.new('\''localhost'\'', 9092) rescue exit(1)'
```